.github/workflows/_buildpacks-release.yml

```diff
@@ -71,14 +71,33 @@ jobs:
         with:
           submodules: true
 
-      - name: Install musl-tools
-        run: sudo apt-get install musl-tools --no-install-recommends
+      - name: Install Languages CLI
+        uses: heroku/languages-github-actions/.github/actions/install-languages-cli@main
+        with:
+          branch: ${{ inputs.languages_cli_branch }}
+          update_rust_toolchain: false
+
+      - name: Generate buildpack matrix
+        id: generate-buildpack-matrix
+        run: actions generate-buildpack-matrix --package-dir ${{ env.PACKAGE_DIR }}
 
       - name: Update Rust toolchain
         run: rustup update
 
-      - name: Install Rust linux-musl target
-        run: rustup target add x86_64-unknown-linux-musl
+      - name: Install cross-compile tooling
+        env:
+          RUST_TRIPLES: ${{ steps.generate-buildpack-matrix.outputs.rust_triples }}
+        run: |
+          for triple in $(jq --exit-status -r '.[]' <<< "${RUST_TRIPLES}"); do
+            if [[ "$triple" == "aarch64-unknown-linux-musl" ]]; then
+              sudo apt-get install musl-tools gcc-aarch64-linux-gnu g++-aarch64-linux-gnu libc6-dev-arm64-cross --no-install-recommends
+            elif [[ "$triple" == "x86_64-unknown-linux-musl" ]]; then
+              sudo apt-get install musl-tools --no-install-recommends
+            else
+              echo "WARNING: Unknown triple: $triple. Cross-compile dependencies were skipped."
```

Comment:
Mostly for any bash buildpacks (`heroku/nodejs-npm`). They'll have a target of `linux` (with no architecture), so we won't have a rust triple for that. We don't need a rust triple for that either, since we don't run `cargo libcnb package` on it.

Reply:
I've changed the rust logic that generates the list of rust triples. It will now error in any case where a rust triple is expected (libcnb.rs bulidpacks) and the target doesn't have a known triple. This else has been removed.